ecs-proxies-deploy.yml

```diff
@@ -1,13 +1,17 @@
-docker_service_cpu: 1024
-docker_service_memory: 2048
+docker_service_cpu: 2048
+docker_service_memory: 4096
+docker_service_autoscaling_prod:
+    service_metric: ECSServiceAverageCPUUtilization
+    target_value: 50
```

Comment:
Just a general comment to keep a note of these values and how they change with each execution so we have a history